pkg/security/probe/process_killer_linux.go

```diff
@@ -20,6 +20,18 @@ const (
 	userSpaceKillWithinMillis = 2000
 )
 
+var (
+	// list of binaries that can't be killed
+	binariesExcluded = []string{
+		"/opt/datadog-agent/bin/agent/agent",
+		"/opt/datadog-agent/embedded/bin/trace-agent",
+		"/opt/datadog-agent/embedded/bin/security-agent",
+		"/opt/datadog-agent/embedded/bin/process-agent",
+		"/opt/datadog-agent/embedded/bin/system-probe",
+		"/opt/datadog-agent/bin/datadog-cluster-agent",
+	}
+)
```

Comment:
Out of curiosity, is there a downside to using prefixes (`/opt/datadog-agent/bin` + `/opt/datadog-agent/embedded/bin`) instead of the specific executables since they should be `dd-agent` owned dirs and would be a bit more future-proof?

Reply:
I think we should keep the list as precise as possible. There is a lot of process that could beused by an attacker in the directories, i.e `curl`.